python/tvm/autotvm/measure/measure_methods.py

```diff
@@ -227,7 +227,8 @@ def set_task(self, task):
 
     def get_build_kwargs(self):
         kwargs = {}
-        if 'cuda' in self.task.target.keys or 'opencl' in self.task.target.keys:
+        if 'cuda' in self.task.target.keys or 'opencl' in self.task.target.keys or \
+           'rocm in self.task.target.keys':
```

Comment:
'rocm'

Reply:
Ohoh.

Reply:
Thanks for spotting this, I fixed it. (It did work remarkably well, that trap of overly loose typing.)

Reply:
I noticed that CI was red, it was complaining that there was a type error :)

Reply:
Maybe the first time I saw python type checker in action